openhub/management/commands/import_portfolio_projects_data.py

```diff
@@ -0,0 +1,12 @@
+from openhub.data import get_data
+from openhub.portfolio_projects import import_data
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Import PortfolioProjects data'
+
+    def handle(self, *args, **options):
+        data = 'projects'
```

Comment:
put this string into a class level CONSTANT with a more meaningful variable name